lighthouse-core/gather/driver.js

```diff
@@ -596,6 +619,26 @@ class Driver {
   }
 
   /**
+   * @return {!Promise<number>}
+   */
+  _getOrCreateIsolatedContextId() {
```

Comment:
maybe a jsdoc comment here that the contextId won't persist past a page load (so should be cleared and recreated)?

Reply:
good call, done